client/state/sites/test/selectors.js

```diff
@@ -2600,6 +2601,96 @@ describe( 'selectors', () => {
 		} );
 	} );
 
+	describe( 'getJetpackSearchDashboardUrl()', () => {
+		test( 'should return null if no sites loaded', () => {
+			const dashboardUrl = getJetpackSearchDashboardUrl(
+				{
+					sites: {
+						items: {},
+					},
+				},
+				2916284
+			);
+
+			chaiExpect( dashboardUrl ).to.be.null;
```

Comment:
We're using Jest assertions these days rather than Chai in new tests. You can easily change them:

`expect( dashboardUrl ).toBeNull();`

Reply:
Tests have been updated!